doc/reference/security.rst

```diff
@@ -257,6 +257,64 @@ While formatting the template differently would prevent such vulnerabilities.
         font-weight: bold;
     }
 
+Escaping vs Sanitizing
+----------------------
+
+**Escaping** converts `TEXT -> CODE`. It is absolutely mandatory to do it every
```

Comment:
```suggestion
**Escaping** converts TEXT to CODE. It is absolutely mandatory to do it every
```

Reply:
I was wondering about that. The change of style increases the importance. Could be bold but "Escaping" is already in bold...